samples_test/src/tests/algorithms.rs

```diff
@@ -258,6 +258,26 @@ pub const SIMPLEISING_EXPECT: Expect =
     expect!["[Zero, Zero, Zero, One, One, Zero, One, One, Zero]"];
 pub const SIMPLEISING_EXPECT_DEBUG: Expect =
     expect!["[Zero, Zero, Zero, One, One, Zero, One, One, Zero]"];
+pub const SIMPLEVQE_EXPECT: Expect = expect![[r#"
+    Beginning descent from value 0.9795.
+    Value improved to 0.9511000000000001.
+    Value improved to 0.8876000000000001.
+    Value improved to 0.8764000000000001.
+    Value improved to 0.8702000000000001.
+    Value improved to 0.8641000000000002.
+    Value improved to 0.8549000000000001.
+    Descent done. Attempts: 100, Step: 0.00000037450702829239286, Arguments: [1.5886651273511148, 1.3926990816987241], Value: 0.8549000000000001.
+    0.8549000000000001"#]];
+pub const SIMPLEVQE_EXPECT_DEBUG: Expect = expect![[r#"
```

Comment:
What's the difference between this and the non-debug version above?

Reply:
There's no difference. Just like in most other sample tests (if not all of them).
When we autogenerate Rust test cases [here](https://github.com/microsoft/qsharp/blob/740c4923bc047adbbb68aa81171dad5bee6b1235/samples_test/build.rs#L63), we include both 'run' and 'debug' version of the test. So both strings are needed with our current approach to sample testing.